src/components/LiquidityMining/Create/Steps/Time/index.tsx

```diff
@@ -1,84 +1,122 @@
+    (newStartTime: Date) => {
+      if (endTime ? newStartTime.getTime() > endTime.getTime() : Date.now() > newStartTime.getTime()) return
+      // if (Date.now() > newStartTime.getTime() || (endTime && endTime.getTime() <= newStartTime.getTime())) return // date in the past, invalid
+      setStartTime(newStartTime)
+    },
+    [setStartTime, endTime]
+  )
+
+  const handleEndTimeChange = useCallback(
+    (newEndTime: Date | null) => {
+      if (!newEndTime) {
+        setEndTime(null)
+        return
+      }
+      if (startTime ? startTime.getTime() > newEndTime.getTime() : Date.now() > newEndTime.getTime()) return // date in the past, invalid
+      setEndTime(newEndTime)
+    },
+    [startTime, setEndTime]
+  )
   return (
-    <Card>
-      <FlexContainer justifyContent="stretch" width="100%">
-        <Box flex="1" data-testid="start-time-selector-box">
-          <TimeSelector
-            title="START DATE AND TIME"
-            placeholder="Start date & time"
-            value={startTime}
-            minimum={new Date()}
-            onChange={onStartTimeChange}
-          />
-        </Box>
-        <Box mx="18px">
-          <Divider />
-        </Box>
-        <ResponsiveBoxContainer flex="1" data-testid="end-time-selector-box">
-          <TimeSelector
-            title="END DATE AND TIME"
-            placeholder="End date & time"
-            value={endTime}
-            minimum={startTime || new Date()}
-            onChange={onEndTimeChange}
-          />
-        </ResponsiveBoxContainer>
-        <Box mx="18px">
-          <Divider />
-        </Box>
-        <ResponsiveFlexContainer flexDirection="column">
-          <Box mb="16px">
-            <TYPE.small fontWeight="600" color="text4" letterSpacing="0.08em">
-              TIMELOCK
-            </TYPE.small>
-          </Box>
-          <Box>
-            <Toggle isActive={timelocked} toggle={onTimelockedChange} />
-          </Box>
-        </ResponsiveFlexContainer>
-      </FlexContainer>
-    </Card>
+    <FlexWrapper>
+      <StyledSmoothGradientCard height="150px" width="413px" padding={'42.5px 28px'}>
+        <TimeSelector
+          data-testid="start-time-selector-box"
+          title="STARTING"
+          placeholder="Start date"
+          value={startTime}
+          minimum={new Date()}
+          onChange={handleStartTimeChange}
+        />
+
+        <HorizontalDivider />
+
+        <TimeSelector
+          data-testid="end-time-selector-box"
+          title="ENDING"
+          placeholder="End date"
+          value={endTime}
+          minimum={startTime || new Date()}
+          onChange={handleEndTimeChange}
+        />
+      </StyledSmoothGradientCard>
+
+      <SmoothGradientCard
+        textAlign={'start'}
+        alignItems={'start !important'}
+        padding={'41px'}
+        width={'299px'}
+        height={'150px'}
+        flexDirection={'column'}
+        justifyContent={'space-around !important'}
+      >
```

Comment:
Something feels wrong to me if we need to define `!important` in this component

Reply:
yeah Ill try to reduce it as much as i can on this gradient card